src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * This UI class handles the output string of Duke
+ * 
+ * @param logo Duke logo
+ * @param line Indentation line
+ * @param terminate_input input to end Duke
+ * @param indentation spaces to aline the output string
+ * 
+ * @param sc Scanner to scan the user input
+ * 
+ * @author WangYihe
+ * @author E0424695
+ */
+
+package duke.ui;
+
+import duke.task.*;
```

Comment:
It might be a better idea to import relevant files from the task folder instead of importing all?